api_gateway/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+# comma-separated list of services
+serviceQueues=
```

Comment:
There should be some mention in documentation that updating this field is required while adding new service